tests/unit/test_cli_parsing.py

```diff
@@ -0,0 +1,88 @@
+"""
+Unit tests for CLI issue body parsing
+"""
+
+import pytest
+from faq_automation.cli import parse_issue_body
+
+
+def test_parse_issue_body_simple():
+    """Test parsing simple issue body"""
+    issue_body = """### Question
+How do I install dependencies?
+
+### Answer
+Run: pip install -r requirements.txt
+"""
+
+    question, answer = parse_issue_body(issue_body)
+
+    assert question == "How do I install dependencies?"
+    assert "pip install -r requirements.txt" in answer
+
+
+def test_parse_issue_body_multiline():
+    """Test parsing issue body with multiline content"""
```

Comment:
What are we doing here? Is it not something we do in JS? Or you're already moving the code to Python?

Reply:
This test verifies that `parse_issue_body` correctly handles multiline content in both the question and answer sections. Just a more real-world test of this function to make sure its issue parsing logic doesn't get tripped up by complex issue inputs from users. It's the `parse_issue_body` function that needs to be used to replace some of the JS code, which I will work on now that I've wrestled with these other issues you pointed out.